date/books.json

```diff
@@ -0,0 +1 @@
+[{"json_class":"Book","title":"Harry Potter","author":"Rowling"},{"json_class":"Book","title":"The Lord of the Rings","author":"Tolkien"},{"json_class":"Book","title":"kjbhjh","author":"kbjkhb"}]
```

Comment:
- [ ] It appears you are not making use of the files within the `date` directory. When I create books, people, and rentals, the files are saved in JSON files within the root path. Kindly remove this file and all the ones within the `date` folder to keep your code clean.